react/week3/todos/src/components/counter.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+class Counter extends React.Component {
+    constructor(props) {
+      super(props)
+      this.state = {
+        value: 0
+      }
```

Comment:
It's not necessary to declare state like this in the version of react we're using: 
```suggestion
     state = {
        value: 0
      }
```

Reply:
It's not clear from the docs, to be fair! React takes care of the constructor stuff behind the scenes for us.